starters/docs/src/Toolbar.css

```diff
@@ -0,0 +1,50 @@
+@import './Checkbox.css';
+@import './Button.css';
+@import './ToggleButton.css';
+@import "./theme.css";
+
+.react-aria-Toolbar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 5px;
+
+  &[data-orientation=horizontal] {
+    flex-direction: row;
+  }
+
+  .react-aria-Group {
+    display: contents;
+  }
+
+  .react-aria-ToggleButton {
+    width: 32px;
+  }
+}
+
+.react-aria-Separator {
```

Comment:
wonder if this should be specific to toolbar or pulled out into a general Separator component?

Reply:
I think i'd opt to leave it as is for the moment, we didn't reuse it in S2 as much either and none of the other stories/components are currently using this